miner/worker.go

```diff
@@ -111,6 +111,7 @@ const (
 type newWorkReq struct {
 	interrupt *int32
 	noempty   bool
+	tstamp    int64
```

Comment:
One more nitpick, please rename this to timestamp (also the one in commitNewWork). There's really no value in leaving out 3 characters.